content/ecosystem/data.toml

```
@@ -854,6 +854,11 @@ name = "sfxr"
 source = "crates"
 categories = ["audio"]
 
+[[items]]
+name = "shaderc"
+source = "crates
```

There's a closing quote missing here: 

```suggestion
source = "crates"
```